dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -1044,6 +1044,9 @@ else if (paramType.equals(Character.class)) // special case: Character
 
             else if(paramType.equals(char[].class)) // special case: char array
                 param = value.toCharArray();
+            
+            else if (paramType.isEnum()) // special case: Enum
+                param = Enum.valueOf((Class<Enum>)paramType, value.toUpperCase());
```

Comment:
Should we add some exception handling here? 
It is possible enum values could be removed in future driver updates which will result in an `IllegalArgumentException` being thrown. We should handle that exception and perhaps add a warning that `this version of the JDBC driver doesn't support value x`